backend/tests/v5/unit/middleware/dataConverter/inputs/teamspaces/settings.test.js

```diff
@@ -419,8 +423,8 @@ const testCheckTicketTemplateExists = () => {
 			expect(Responder.respond).not.toHaveBeenCalled();
 		});
 
-		test(`Should respond with ${templates.templatesNotFound} if template does not exist`, async () => {
-			TemplateModelSchema.getTemplateById.mockRejectedValueOnce(templates.templatesNotfound);
+		test(`Should respond with ${templates.templateNotFound} if template does not exist`, async () => {
```

Comment:
I think it needs to be this?

```suggestion
		test(`Should respond with ${templates.templateNotFound.code} if template does not exist`, async () => {
```